lib/view/seasons/seasons_screen.dart

```diff
@@ -44,13 +45,14 @@ class _SeasonsPageState extends State<SeasonsPage> {
 
   FloatingActionButton floatingActionButton(BuildContext context) {
     return FloatingActionButton(
-      onPressed: () async =>onPressedFloating(),
+      onPressed: () async => onPressedFloating(),
       child: const Icon(Icons.add),
     );
   }
-  onPressedFloating(){
-    Navigator.push(
-        context, MaterialPageRoute(builder: (context) => const AddSeasonItem()));
+
+  onPressedFloating() {
```

Comment:
"onPressedFloating" gives no information about what the function actually does.
The function goes to the AddSeasonItem screen.
You can, for example, name it "pushAddSeasonItemScreen"

Reply:
solved #14